client/components/screen-options-tab/index.js

```diff
@@ -86,6 +89,13 @@ const ScreenOptionsTab = ( { wpAdminPath } ) => {
 	}
 
 	const onSwitchView = ( view ) => {
+		recordTracksEvent( 'wpcom_dashboard_quick_switch_link_clicked', {
```

Comment:
The event is not being tracked right now, I think because this is lacking a `dispatch` call:

```js
dispatch( recordTracksEvent( 'wpcom_dashboard_quick_switch_link_clicked', { // ... } ) );
```